### ISSUE TITLE
feature request: compressing the compressed list a bit more

### ISSUE THREAD

**Issue by mrwth** [mb1986/rm-hacks#339]
In the ddvk's hacks the "compressed view" showed folders in a grid (the default one, which is quite compact when it comes to folders) and files in a compressed list, just as in the rm-hacks. Could we have this view in the rm-hacks?  It is/was more compact than the current rm-hacks' compressed list.

**Comment by PepikVaio** [comment]
Can you post a photo?

**Comment by WieValdi** [comment]
@mrwth is it this what you are thinking about? https://github.com/ddvk/remarkable-hacks/issues/388#issuecomment-1016941017

**Comment by mrwth** [comment]
@WieValdi yes, that's what it looked like for folders, below them it continued as the compressed list of files (I don't have a screenshot since I switched to the current software and rm-hacks)
